environment/src/main/java/com/sequenceiq/environment/environment/service/AbstractEnvironmentAwareService.java

```diff
@@ -43,6 +43,10 @@ public T create(T resource, String accountId) {
         return conversionService.convert(attachToEnvironments(resourceName, environments, workspaceId), classToConvert);
     }
 
+    public <C> C attachToEnvironmentsAndConvert(String resourceName, Set<String> environments, @NotNull String accountId, Class<C> classToConvert) {
```

Comment:
Annotating the accountId field with `@NotNull` is not necessary since we're not calling any functions on this field later on.

Reply:
It was bring from cloudbreak "as is" not in this change. I would not change it.